### PR TITLE
Rails 4 fix quotas bulk update

### PR DESCRIPTION
**Pull request by Levia**
## Description

It seems that this version of Rails 4 has got a bug with regards casting years extracted from postgres dates through ActiveRecord, resulting into floating numbers ([more about this here](https://www.reddit.com/r/rails/comments/6rxsxe/extract_year_from_created_atdate_giving_me_the/))

I've added the necessary casting to every 'EXTRACT' statement.
Also, I fixed a bug regarding the counts of quotas that are going to be duplicated. The initial count wasn't taking into consideration only current quotas.

## Notes

[Codebase ticket](https://unep-wcmc.codebasehq.com/projects/species-rails-4-upgrade/tickets/52)

### CODE REVIEW

File: app/controllers/admin/quotas_controller.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ def index
 
   def duplication
     @years = Quota.years_array
-    @count = Quota.where('EXTRACT(year from start_date) = ?', @years.first).
+    @count = Quota.where('EXTRACT(year from start_date)::VARCHAR = ? AND is_current = true', @years.first).
```
Comment (lucacug) [inline app/controllers/admin/quotas_controller.rb:13]: Tha date cast is working for me, just out of curiosity, the wrong counting (so the add of the is_current = true bit) is not related to the rails4 upgrade, is it?

Comment (Levia) [reply app/controllers/admin/quotas_controller.rb:13]: That's right. I noticed that and it looked like a bug, so fixed it too :)

**Comment by lucacug** [comment]
@Levia Happy with the code, again we got a conflict here

**Comment by Levia** [comment]
@lucacug Pushed another commit to also fix Travis build

**Pull request merged**